docs/tools/pow-vs-pos-vs-dpos.md

```diff
@@ -0,0 +1,63 @@
+---
+id: pow-vs-pos-vs-dpos
+title: PoW vs PoS vs DPoS
+sidebar_label: PoW vs PoS vs DPoS
+---
+
+It is necessary to know the consensus mechanisms used by the different blockchain networks to maintain security and integrity within the network, as well as the benefits and the networks that still conform it.
```

Comment:
It is necessary to know ...

replace with 

It is important to understand ...

Reply:
Fixed up